docs/src/content/docs/guides/components.mdx

```diff
@@ -91,7 +91,7 @@ The code above generates the following tabs on the page:
 
 Keep multiple tab groups synchronized by adding the `syncKey` attribute.
 
-All `<Tabs>` on a page with the same `syncKey` value will display the same active label. This allows your reader to choose once (e.g. their operating system or package manager), and see their choice reflected throughout the page.
+All `<Tabs>` with the same `syncKey` value will display the same active label. This allows your reader to choose once (e.g. their operating system or package manager), and see their choice persisted using [`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) and reflected throughout the entire website.
```

Comment:
Do you think mentioning the implementation detail of local storage is important here? Might just be a distraction and behaviour is clear enough with this:

```suggestion
All `<Tabs>` with the same `syncKey` value will display the same active label. This allows your reader to choose once (e.g. their operating system or package manager), and see their choice persisted across page navigations.
```